.github/workflows/test.yml

```diff
@@ -139,6 +139,13 @@ jobs:
     steps:
       - uses: RDXWorks-actions/checkout@main
 
+      - name: Remove old xcode versions
```

Comment:
We can try this.

But it is possible this breaks test Swift - and possible it works for most runs but fails for some due to non determinism.

If we have troubles we can change so that Kotlin tests are run AFTER Swift tests:

Which can be done by using `needs: test-Swift`